### ISSUE TITLE
[REQUEST] free ram after closing a job

### ISSUE THREAD

**Issue by APT-ZERO** [openbullet/OpenBullet2#386]
i was using OB2 latest update
i loaded two huge wordlists for two jobs
then ram usage went on 700mb (for two 10gb+ wordlist)
after days, i closed both jobs, but ram was still same as before i closed that jobs
then to free ram, i had to end dotnet.exe process and run OB2 again

ram usage was good and there is no problem, even it was amazing that OB2 can handle huge wordlists with that little of ram
but after i closed both jobs, when there is no need for that wordlist or anything else anymore, shouldn't OB2 free the ram?

**Comment by openbullet** [comment]
Yeah there are some references somewhere that I cannot find and they are preventing the garbage collector from freeing up the heap. I will investigate.

> it was amazing that OB2 can handle huge wordlists with that little of ram

This is because I do not load the entire wordlist in ram but I read it from disk line by line when needed.

**Comment by openbullet** [comment]
Fixed
